clab/authz_keys.go

```diff
@@ -18,24 +18,24 @@ import (
 )
 
 const (
-	pubKeysGlob = "~/.ssh/*.pub"
+	pubKeysGlobPart = "~/.ssh/*.pub"
 	// authorized keys file path on a clab host that is used to create the clabAuthzKeys file.
-	authzKeysFPath = "~/.ssh/authorized_keys"
+	authzKeysFPathPart = "~/.ssh/authorized_keys"
```

Comment:
These are not `...Part` anymore I presume

Reply:
sure yes